app/views/partials/_template_editable_collection_item_menu.html.erb

```diff
@@ -0,0 +1,10 @@
+<script type="text/html"
+        data-component-template="EditableCollectionItemMenu"
+        data-activator-text="<%= t('question.menu.activator') %>">
+  <ul class="govuk-navigation component-activated-menu">
+    <li data-action="remove"
+        data-api-path="<%= URI.unescape( api_service_page_question_question_option_destroy_message_path(service.service_id,@page.uuid,'#{question_uuid}', '#{option_uuid}', method: :delete) ) -%>">
```

Comment:
i don't think we need to unescape this? rails `_path` methods should output a safe to use URL

Reply:
Yeah, I wasn't 100% sure about this - should have mentioned it on our call. The unescape is here because I actually don't want a safe to use url at this point. Rails escapes the placeholders `#{quesiton_uuid}` so without this the string that gets to JS includes `%23%7Bquestion_uuid%7D%0A`

I guess I could remove the `URI.unescape` and unencode the url in the js instead.... I just figured that as the string in the data-attribute doesn't need to be a valid url, then its not a problem if it contains unencoded characters. The JS is then responsible for replacing the placeholders and creating a valid url from it.

Happy to change though if you'd prefer?